src/reanimated2/utils.ts

```diff
@@ -14,6 +14,9 @@ export function getRelativeCoords(
 ): ComponentCoords {
   'worklet';
   const parentCoords = measure(parentRef);
+  if (!parentCoords) {
```

Comment:
Let's be more explicit.
```suggestion
  if (parentCoords !== null) {
```

Reply:
I made an equivalent change (the suggested change would actually have inverted the check), but good call. 👍

Reply:
Oops, my fault indeed. Now it's good 👍